src/assets/css/main.scss

```diff
@@ -0,0 +1,18 @@
+@import 'normalize.css';
+
+$breakpoints: (
```

Comment:
Create a separate `_breakpoint.scss` file which will then be used by rest of the files.